clusterloader2/pkg/measurement/util/histogram.go

```diff
@@ -28,6 +34,34 @@ type Histogram struct {
 	Buckets map[string]int    `json:"buckets"`
 }
 
+// Quantile calculates the quantile 'q' based on the given buckets of Histogram.
+func (h *Histogram) Quantile(q float64) (float64, error) {
+	hist := testutil.Histogram{
+		&dto.Histogram{
+			Bucket: []*dto.Bucket{},
+		},
+	}
+
+	for k, v := range h.Buckets {
```

Comment:
please add a comment clarifying that hist.Quantile expected a slice of Buckets ordered by UpperBound.

Reply:
Done, right before calling sort.Slice